core/src/edu/cornell/gdiac/main/FanModel.java

```diff
@@ -56,6 +56,28 @@ public FanModel() {
     wind = new WindModel();
   }
 
+  public void setPeriod(float period){
+    this.period = period;
+  }
+
+  public void setPeriodOnRatio(float periodOnRatio){
+    this.periodOnRatio = periodOnRatio;
+  }
+
+  public void setWindParticleFixtures(int numWindParticles){
+    windParticleFixtures = new Fixture[numWindParticles];
+  }
+
+  public void setFanSide(WindSide windSide){
+    fanSide = windSide;
+  }
+
+  public void setWind(WindModel wind){
+    this.wind = wind;
+  }
+
+
+
```

Comment:
None of these methods such exist since we want initialization to happen within the object model